### PR TITLE
Advance cursor to year after month entry

### PR DESCRIPTION
**Pull request by rmayhue**

### CODE REVIEW

File: date-input.html [hunk 1]
```diff
@@ -51,6 +51,7 @@
 
     <div class="horizontal layout">
       <input is="iron-input"
+          id="expMonth"
```
Comment (notwaldorf) [inline date-input.html]: This doesn't seem to be used anywhere

File: date-input.html [hunk 2]
```diff
@@ -69,6 +70,7 @@
           readonly$="[[readonly]]">
       <span>/</span>
       <input is="iron-input"
+          id="expYear"
```
Comment (notwaldorf) [inline date-input.html]: I would rename this to `dateYear` or `expirationYear`. Abbreviations are icky :)

**Comment by notwaldorf** [comment]
Awesome contribution! I just have a couple of super minor comments :)

File: date-input.html (hunk 1 above)
Comment (rmayhue) [inline date-input.html]: No it's not.  I just added the id on the month input when I added the id to the year input. I can remove it.

File: date-input.html (hunk 2 above)
Comment (rmayhue) [inline date-input.html]: Agreed. I was going by the autocomplete="cc-exp-year". I'll change it to expirationYear. Seems a bit more descriptive the dateYear.

**Comment by notwaldorf** [comment]
Thanks! Would you mind squashing your commits, please?

**Pull request merged**